assets/data/effects/wilderness/encounter_wilderness_familyBusiness_revision.json

```diff
@@ -0,0 +1,13511 @@
+							"textKey": "~01~prompt~panel_028~4_hero",
+							"pointerOwner": "hero",
+							"padXFraction": 0.233,
+							"padYFraction": 0.002,
+							"shrinkToFitText": true
+						},
+						{
+							"textSourceFile": "wilderness/encounter_wilderness_familyBusiness_revision",
+							"textKey": "~01~prompt~panel_028~5_volunteer",
+							"pointerOwner": "volunteer",
+							"location": "bottomLeft",
+							"padXFraction": 0.283,
+							"padYFraction": 0.141,
+							"widthFraction": 0.6657664,
+							"shrinkToFitText": true,
+							"pointerAdjust": { "rotate": -2.7903144, "dX": -18.734623, "dY": -2.1617317 }
+						}
+					]
+				}
+			},
+			{
+				"class": "comicNode_Panel",
+				"panel": {
+					"showNameTags": false,
+					"bg": "comicBG_tavern.jpg",
+					"actorSlots": [
+						{
+							"role": "volunteer",
+							"form": "frame-grim",
+							"equipment": {
+								"mode": "removeWeapons",
+								"equipItemsById": [ "mug" ]
+							},
+							"anchor": { "x": -1.694, "y": 0.604 },
+							"size": 1.0551383,
+							"drawDepth": 0.25,
+							"nameTag": "bottomRight",
+							"head": { "rotate": -4.106 },
+							"color": { "r": "0.73504406", "g": "0.65441132", "b": "0.52372891", "a": "1" }
+						},
+						{
+							"role": "hero",
+							"form": "frame-grim",
+							"facing": "left",
+							"equipment": {
+								"mode": "removeWeapons",
+								"equipItemsById": [ "mug" ]
+							},
+							"anchor": { "x": -0.741, "y": 0.554 },
+							"size": 0.77385,
+							"head": { "rotate": 4.501, "scaleY": 1 },
+							"color": { "r": "0.73504406", "g": "0.65441132", "b": "0.52372891", "a": "1" }
+						},
+						{
+							"role": "npc",
+							"form": "frame-grim",
+							"facing": "left",
+							"equipment": {
+								"mode": "removeWeapons",
+								"equipItemsById": [ "mug" ]
+							},
+							"anchor": { "x": 0.829, "y": 0.62 },
+							"size": 1.0215267,
+							"drawDepth": -2,
+							"nameTag": "none",
+							"head": { "rotate": -12.313 },
+							"color": { "r": "0.73504406", "g": "0.65441132", "b": "0.52372891", "a": "1" }
+						},
+						{
+							"role": "npc2",
+							"form": "frame-skeptical",
+							"facing": "left",
+							"equipment": {
+								"mode": "removeWeapons",
+								"equipItemsById": [ "mug" ]
+							},
+							"anchor": { "x": 2.992, "y": 0.384 },
+							"size": 0.6551484,
+							"nameTag": "none",
+							"head": { "rotate": 2.714 },
+							"foot": { "rotate": 5.041 },
+							"color": { "r": "0.73504406", "g": "0.65441132", "b": "0.52372891", "a": "1" }
+						},
+						{
+							"role": "npc3",
+							"form": "frame-skeptical",
+							"equipment": { "mode": "removeWeapons" },
+							"anchor": { "x": 2.218, "y": 0.395 },
+							"size": 0.6094766,
+							"head": { "rotate": 10.767, "scaleY": 1 },
+							"color": { "r": "0.73504406", "g": "0.65441132", "b": "0.52372891", "a": "1" }
+						},
+						{
+							"role": "npc4",
+							"form": "frame-scared",
+							"equipme
```

Comment:
🤔 Aye, it would be nice if we could pass `hero` in as a parameter or as the value or something so that we could deliberately choose some major details to match their relative...